### DISCUSSION TITLE
File 'mybook-p335.m4b' already exists. Overwrite? [y/N] Not overwriting ... Are my files all gone?

### DISCUSSION THREAD

**Discussion by Nikanoru** [aedocw/epub2tts#174] [Q&A] [answered]
Sorry! It's me again.

This morning, once I got epub2tts working, I test ran 1 chapter of an epub to see if it sounds fine and after I verified that the m4b file sounded fine to me, I let the program run through the entire 400+ page epub.

I went away from my computer, and when it finished I see this in my powershell window:

File 'mybook-p335.m4b' already exists. Overwrite? [y/N] Not overwriting - exiting
100.00% spoken so far.
Elapsed: 261 minutes, ETA: 0 minutes
mybook-p335.m4b complete

In the folder, there is only the m4b of the 1 chapter test run. I really hope my computer did not spent the last 4 hours in vain and discarded the entire epub m4b because I did not select overwrite in time and the program automatically discarded the entire work^^ but I can not seem to find them and I don't know where to look for them/the file.

**Comment by aedocw** [comment, marked as answer]
Unfortunately it looks like your computer wasted your time. I'm surprised about something timing out, there is no timer or anything, it should have just sat and waited patiently for your response.

It would be good to move that warning to the START of epub2tss though, to check to see if the ultimate output file exists before doing any work. I'll add that issue as an enhancement.

**Reply by Nikanoru** [reply]
That sounds like a smart idea to move the warning to the start hahaha :D

No worries, thanks for the quick response! I'll just redo it later then and make sure to delete the test file beforehand.

**Comment by aedocw** [comment]
https://github.com/aedocw/epub2tts/issues/175

**Comment by aedocw** [comment]
Thanks for bringing this up, this fix was a really good improvement to the script. Sorry you had to find out about it the hard way haha!
